data/publiccloud/xfstests/partitions.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+disk=$1
+
+parted -s /dev/${disk} mklabel gpt
+parted -s /dev/${disk} mkpart primary 0% 50%
+parted -s /dev/${disk} mkpart primary 50% 100%
+
+mkfs.xfs /dev/${disk}1
```

Comment:
/dev/${disk}1 and 2 depend on init status for the disk. I'm not sure this gonna work in all disk or not. Better to check before use.

Reply:
the status will be always new and unpartitioned and unformatted. So, I guess we will have always sdX1 and sdX2